notebooks/225-yolov7-optimization/225-yolov7-optimiization.ipynb

```diff
@@ -0,0 +1,706 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Convert Yolo V7 to ONNX and OpenVINO™ IR\n",
+    "\n",
+    "This short tutorial demonstrates step-by-step instruction how to convert Pytorch Yolo V7 to OpenVINO IR. \n",
```

Comment:
It's not a "short tutorial", for sure)
I would rephrase it to:
```
This tutorial demonstrates step-by-step instructions on how to run and optimize *PyTorch Yolo V7 with OpenVINO.

It consists of the following steps:
```

Reply:
Updated